virttest/utils_misc.py

```diff
@@ -2791,7 +2791,7 @@ def format_windows_disk(session, did, mountpoint=None, size=None,
             logging.debug("Detail for 'Disk%s'" % did)
             details = session.cmd_output(detail_cmd)
 
-            if re.search("Status.*Offline", details, re.I | re.M):
+            if re.search("Disk.*Offline", details, re.I | re.M):
```

Comment:
Yes, it is. And I also ran this job on win7, win8, win10 and win2012, it can be passed on all guests. 

PASS 3-smp_2.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_scsi.up.virtio_net.Win7.i386.sp1.io-github-autotest-qemu.block_resize
PASS
4-smp_2.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_scsi.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.block_resize

 (06/10) smp_2.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_scsi.up.virtio_net.Win10.x86_64.io-github-autotest-qemu.block_resize: PASS (482.31 s)
 (07/10) smp_2.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_scsi.up.virtio_net.Win2012.x86_64.io-github-autotest-qemu.block_resize: PASS (232.71 s)
 (08/10) smp_2.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_scsi.up.virtio_net.Win2012.x86_64.r2.io-github-autotest-qemu.block_resize: PASS (200.04 s)
 (09/10) smp_2.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.io-github-autotest-qemu.block_resize: PASS (269.64 s)
 (10/10) smp_2.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.block_resize: PASS (238.49 s)